src/NovelRT.DotNet/Interop/Animation/NovelRT.cs

```diff
@@ -7,108 +7,87 @@ namespace NovelRT.Interop
 {
     public static unsafe partial class NovelRT
     {
-        [DllImport("Interop", ExactSpelling = true)]
-        [return: NativeTypeName("NrtResult")]
-        public static extern int Nrt_SpriteAnimator_create([NativeTypeName("NrtNovelRunnerHandle")] IntPtr runner, [NativeTypeName("NrtImageRectHandle")] IntPtr rect, [NativeTypeName("NrtSpriteAnimatorHandle *")] IntPtr* outputAnimator);
+        [DllImport("NovelRT.Interop", ExactSpelling = true)]
+        public static extern NrtResult Nrt_SpriteAnimator_create([NativeTypeName("NrtNovelRunnerHandle")] IntPtr runner, [NativeTypeName("NrtImageRectHandle")] IntPtr rect, [NativeTypeName("NrtSpriteAnimatorHandle *")] IntPtr* outputAnimator);
```

Comment:
There are three changes from regenerating the bindings:
1. The `DllImport` has changed from `Interop` to `NovelRT.Interop`
2. Where an enum was being taken or returned, we now take return the actual `enum` rather than the `int` it was typedefd to
3. Where `NativeTypeName` and the managed type name differed only in whitespace, the attribute was removed